NineChronicles.Headless.Executable/Program.cs

```diff
@@ -218,10 +218,33 @@ static void ConfigureSentryOptions(SentryOptions o)
 
             Log.Logger = loggerConf.CreateLogger();
 
-            var tasks = new List<Task>();
+            if (appProtocolVersionToken is null)
+            {
+                throw new CommandExitedException(
+                    "--app-protocol-version must be present.",
+                    -1
+                );
+            }
```

Comment:
In history, `appProtocolVersionToken` was nullable because *libplanet node* mode was optional and it was executed through HTTP API. But, the `appProtocolVersionToken` has been required since #420 merged. So how about change it into non-nullable type instead?

Reply:
Added. https://github.com/planetarium/NineChronicles.Headless/pull/481/commits/19b87259e7df2b0d4f2d676410c46d37d4fa60b4